core/block/editor.go

```diff
@@ -639,6 +639,36 @@ func (s *Service) ModifyDetails(objectId string, modifier func(current *types.St
 	})
 }
 
+func (s *Service) ModifyDetailsList(req *pb.RpcObjectListModifyDetailValuesRequest) (resultError error) {
+	var anySucceed bool
+	for _, objectId := range req.ObjectIds {
+		err := s.ModifyDetails(objectId, func(current *types.Struct) (*types.Struct, error) {
+			for _, op := range req.Operations {
+				if op.Set != nil && pbtypes.ValueToInterface(op.Set) != nil {
+					// Set operation has higher priority than Add and Remove, because it modifies full value
+					current.Fields[op.RelationKey] = op.Set
+					continue
+				}
+				pbtypes.AddValue(current, op.RelationKey, op.Add)
```

Comment:
maybe use switch / if between Set/Add/Remove operations? It will be clearer

Reply:
I did not get it :see_no_evil: 
Can you please suggest how we can do it via switch